### PR TITLE
Update vocab for new aat terms

### PR DESCRIPTION
**Pull request by azaroth42**
* update aat terms for the new ones (not yet published, coming this month to the web)
* Merge in _briefText and sourceStatement
* Add sci to the context

### CODE REVIEW

File: cromulent/vocab.py [hunk 1]
```diff
@@ -77,33 +77,43 @@ def register_instance(name, data):
...
+	"BibliographyStatement": {"parent": LinguisticObject, "id": "300026497", "label": "Bibliography Statement", "brief": True},
+	"ExhibitionStatement": {"parent": LinguisticObject, "id":"300435424", "label":"Exhibition Statement", "brief": True},
+
+	"BiographyStatement": {"parent": LinguisticObject, "id":"300435422", "label": "Biography Statement", "brief": True},
+	"EditionStatement": {"parent": LinguisticObject, "id":"300435435", "label": "Edition Statement", "brief": True},
+	"PaginationStatement": {"parent": LinguisticObject, "id":"300435440", "label": "Pagination Statement", "brief": True},
+	"FoliationStatement": {"parent": LinguisticObject, "id":"300435441", "label": "Foliation Statement", "brief": True},
+	"LanguageStatement": {"parent": LinguisticObject, "id": "300435433", "label": "Language Statement", "brief": True},
+
+	"TranscribedInscription": {"parent": LinguisticObject, "id": "300028702", "label": "Transcribed Inscription", "brief": True},
```
Comment (kasei) [inline cromulent/vocab.py]: Hard to tell from this visual diff. Is this both a reorganization and an addition? Or are there also changes to existing classes?

Comment (azaroth42) [reply cromulent/vocab.py]: Sorry, I reordered into hopefully more obvious groups (description of the object, description of activity, description of non-object, transcription vs description) _and_ changed several of the ids to match Patricia's new terms.

File: cromulent/vocab.py [hunk 2]
```diff
@@ -77,33 +77,43 @@ def register_instance(name, data):
...
-	"SourceStatement": {"parent": LinguisticObject, "id": "300404764", "label": "Source Statement", "brief": True},
+	"_BriefText": {"parent": LinguisticObject, "id": "300418049", "label":"Brief Text"},
 	"Abstract": {"parent": LinguisticObject, "id":"300026032", "label": "Abstract", "brief": True},
-	"Note": {"parent": LinguisticObject, "id":"300027200", "label": "Note", "brief": True},
 	"Annotation": {"parent": LinguisticObject, "id":"300026100", "label":"Annotation"},
 	"Citation": {"parent": LinguisticObject, "id":"300311705", "label": "Citation", "brief": True},
-	"ExhibitionStatement": {"parent": LinguisticObject, "id":"300054766", "label":"Exhibition Statement", "brief": True},
+
+	"CreditStatement": {"parent": LinguisticObject, "id": "300026687", "label": "Credit Statement", "brief": True},
+	"RightsStatement": {"parent": LinguisticObject, "id": "300435434", "label": "Copyright/License Statement", "brief": True},
```
Comment (kasei) [inline cromulent/vocab.py]: While spot checking, I couldn't find this AAT term at http://vocab.getty.edu/aat/300435434. Is there delay between new terms being added and showing up in the web UI?

Comment (azaroth42) [reply cromulent/vocab.py]: Yes, they only load GraphDB every quarter (!) so it's coming but not there yet.

Comment (kasei) [reply cromulent/vocab.py]: (Also seems really unfortunate that not-found URIs like that are returning HTTP 200.

**Pull request merged**